File: lab02_stack/lab02_task02/scripts/subscriber_node.py
#!/usr/bin/env python
import rospy
##TODO: include the library for a msg for an array integer

##TODO: Complete the callback function. Your callback function should compute the sum (1 + 1/2 + 1/3 +... + 1/n) and print the output on the screen

def callback(msg):
    print 'Delete this line'


def listener():

    rospy.init_node('listener', anonymous=True)

    ##TODO: Define your subscriber and link this with the callback function.

    rospy.spin()


if __name__ == '__main__':
    listener()

File: lab02/lab02task02/src/subscriber_node.cpp
#include "ros/ros.h"
//TODO: include the library for a msg for an array integer

//TODO: Complete the callback function. Your callback function should compute the sum (1 + 1/2 + 1/3 +... + 1/n) and print the output on the screen
void subscribeCallback();

int main(int argc, char **argv)
{
    ros::init(argc, argv, "listener");

    //TODO: Define your subscriber and link this with the callback function.

    ros::spin();

    return 0;
}